.github/workflows/release.yml

```diff
@@ -187,6 +187,7 @@ jobs:
       matrix:
         conf: [ "sha256", "murmur64" ]
         db: [ "mongo44", "postgres15" ]
+        flavour: [ "pro-ha" ]
```

Comment:
I don't think this is required at this point. `release.yml` runs for PRs and I feel it is overkill to run this for more than one flavour.